tests/shell/pipeline/commands/external.rs

```diff
@@ -271,3 +271,27 @@ mod tilde_expansion {
         assert_eq!(actual.out, "1~1");
     }
 }
+
+mod external_command_arguments {
+    use super::nu;
+    use nu_test_support::fs::Stub::EmptyFile;
+    use nu_test_support::{pipeline, playground::Playground};
+    #[test]
+    fn expands_table_of_primitives_to_positional_arguments() {
+        Playground::setup(
+            "expands_table_of_primitives_to_positional_arguments",
+            |dirs, sandbox| {
+                sandbox.with_files(vec![EmptyFile("jonathan_likes_cake.txt")]);
+
+                let actual = nu!(
+                cwd: dirs.test(), pipeline(
+                r#"
+                nu --testbin cococo $(ls | get name)
```

Comment:
This is cool, I like it.

One nit: can we make this test multiple files, so we can ensure that we're taking the whole list and passing it to the test echo?

Reply:
Yeah, I will do it, it will be better with more than one file, although I'm having a bit of problem with `nu --testbin cococo`.
It seems that when I pass anything after that command that is separated by space it is in the end joined together by empty character. Is this correct behavior? There is even test for this(env is not important in this test):

https://github.com/nushell/nushell/blob/a36d2a1586fda5f5f70c065e898455527d265512/crates/nu-cli/tests/commands/with_env.rs#L24-L31

If `cococo` is replacement for external `echo` in tests, so we wouldn't have different result per system as it was said on discord then I think it should behave more like normal(non-Nu) echo so `nu --testbin cococo foo bar` should result with `foo bar`. Now it works like something like this with internal echo `echo foo bar | str collect` with result `foobar`. If it is incorrect behavior then I can provide fix for it, otherwise I will dig more into it.

Reply:
Yeah, I don't think it puts a carriage return at the end by default. You'd need to also pass something like `$(char newline)`

Reply:
Modified test so it now tests multiple files. Tried appending `$(char newline)` although it looks like `Outcome` strips CR/LF, I could force it in `cococo` but I decided to let it go as it might create confusion like how it magically appeared.